src/shared/assets/mottagning/nolle_codes.ts

```diff
@@ -0,0 +1,10 @@
+export const NollegruppRoles: { [key: string]: string[] } = {
+	Kod1: ["Apfelstrudel", "Grupp A"],
```

Comment:
I'm guessing these codes will be a bit more secure? :p

Reply:
And if they're going to be defined in the source code (which is a big meh moment), this repository must be made private before we add them in

Reply:
> And if they're going to be defined in the source code (which is a big meh moment), this repository must be made private before we add them in

Yeah, that was the plan. Although I do agree, we should probably make it a bit more secure. Any suggestions?

Reply:
I think the best option would be to store them in a database and the second best to store hashes of them in the source code.
But I think it is fine to have them here.

Reply:
We should at least move these codes to the .env file. In that case they can be accessed from eg process.env.CODE_1

Reply:
That is probably the best option. I guess i forgot about it :p

Reply:
Moved the codes to the .env file now